modules/router/src/admin/index.ts

```diff
@@ -49,6 +49,37 @@ export class AdminHandlers {
       }),
       this.routerAdmin.getMiddlewares.bind(this.routerAdmin),
     );
+    this.routingManager.route(
+      {
+        path: '/router/route-middlewares',
+        action: ConduitRouteActions.GET,
+        description: `Returns the middleware of an app route.`,
+        queryParams: {
+          path: ConduitString.Required,
+          action: ConduitString.Required,
+        },
+      },
+      new ConduitRouteReturnDefinition('GetAppRouteMiddleware', {
+        middleware: [TYPE.String],
+      }),
+      this.routerAdmin.getRouteMiddlewares.bind(this.routerAdmin),
+    );
+    this.routingManager.route(
+      {
+        path: '/router/patch-middleware',
```

Comment:
Either `'/router/patch-middleware'` -> `'/router/patch-middlewares'` or update the getter route. Pick one as long as it's consistent in the external APIs.
Also update descriptions.